src/deepsparse/server/README.md

```diff
@@ -0,0 +1,164 @@
+                                        supplied.
+
+        --help                          Show this message and exit.
+```
+
+### Single Model Inference
+
+Example CLI command for serving a single model for the **question answering** task:
+
+```bash
+deepsparse.server \
+    --task question_answering \
+    --model_path "zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant-aggressive_95"
+```
+
+PipelineClient for making requests to your server:
+```python
+import json
+from typing import List
+import numpy
+import requests
+
+class PipelineClient:
+    """
+    Client object for making requests to the example DeepSparse inference server
+
+    :param address: IP address of the server, default is 0.0.0.0
+    :param port: Port the server is hosted on, default is 5543
+    """
+
+    def __init__(self, address: str ='0.0.0.0', port: str ='5543'):
+
+        self._url = f'http://{address}:{port}/predict'
+        
+    def __call__(self, **kwargs) -> List[numpy.ndarray]:
+
+        """
+        :param kwargs: named inputs to the model server pipeline. e.g. for
+            question-answering - `question="...", context="..."
+
+        :return: json outputs from running the model server pipeline with the given
+            input(s)
+        """
+
+        response = requests.post(self._url, json=kwargs)
+        return json.loads(response.content)
+```
+
+To make a request, initialize the PipelineClient:
+
+```python
+model = PipelineClient()
+inference = model(question="Who is Mark?", context="Mark is Batman.")
+```
+__ __
+### Multiple Model Inference
+To serve multiple models you can easily build a `config.yaml` file. 
+In the sample YAML file below, we are defining two BERT models to be served by the `deepsparse.server` for the question answering task:
+
+```
+models:
+    - task: question_answering
+        model_path: zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none
+        batch_size: 1
+        alias: question_answering/base
+    - task: question_answering
+        model_path: zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/pruned_quant-aggressive_95
+        batch_size: 1
+        alias: question_answering/quantagg95
+```
+You can now run the server with the config file path passed in the `--config_file` argument:
+
+```bash
+deepsparse.server --config_file config.yaml
+```
+
+💡 **PRO TIP** 💡: When your server is running, you can always use the awesome swagger UI that's built into FastAPI to view your model's pipeline `POST` routes. All you need is to add `/docs` at the end of your host URL:
+
+    localhost:5543/docs
+
+![alt text](./img/swagger_ui_1.png)
+
+When the DeepSparse server is up and running you can can send it requests via our PipelineClient object. In order to select which `/predict` route to call, we have updated the PipelineClient to include the model's `alias` from the `config.yaml` file.
```

Comment:
is there a reason these code changes are added here and not to the file(s)?

Reply:
adding the client.py to this directory now.